lib/src/components/NxTag/types.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import {ReactNode, WeakValidationMap} from 'react';
+import * as PropTypes from 'prop-types';
+
+export interface Props {
+  onTagSelect?: (() => void) | null;
+  tagSelected?: boolean;
+  children: ReactNode;
+  tagColor?: string | null;
```

Comment:
Would be better if this was constrained to just the specific color strings that we support.  See `NxButton`'s `variant` type definition for an example of how to do that sort of thing.

Reply:
> Would be better if this was constrained to just the specific color strings that we support. See `NxButton`'s `variant` type definition for an example of how to do that sort of thing.

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/6da37c27f37f78d5204b4227844627b2adb78404